### DISCUSSION TITLE
MPV on Apple M1

### DISCUSSION THREAD

**Discussion by andrewdunndev** [jeffvli/feishin#56] [General]
How are people installing feishin on Apple M1 and setting the mpv path? I'm a new apple user and I don't know how to use the GUI in feishin to navigate to the `/opt/` area that homebrew installs mpv.

**Comment by flightmansam** [comment]
In your terminal use the command `which mpv` the result is the path that you put in the startup dialogue.

**Reply by andrewdunndev** [reply]
Apologies, it's not that I can't find mpv... it's that feishin pops up a dialog to enter the path and I have no idea how to find `/opt` graphically.

**Reply by andrewdunndev** [reply]
Apparently `Command+Shift+G` allows you to type in a path, so I was able to select the mpv path.

I'm not getting any playback, so I'll investigate further.

**Reply by andrewdunndev** [reply]
As a __brand new__ apple user I'm wondering if its possible to run feishin in a way where I can see logs?

**Comment by andrewdunndev** [comment]
As of v0.0.1-aplha6 I can get playback working, however I still needed to do:

```
xattr -r -d com.apple.quarantine /Applications/Feishin.app
```
